content/pt/docs/concepts/cluster-administration/networking.md

```diff
@@ -0,0 +1,250 @@
+faz de uma maneira diferente.
+
+## O modelo de conectividade e rede do Kubernetes
+
+Todo `Pod` obtém seu próprio endereço IP. Isso significa que vocë não precisa 
+criar links explícitos entre os `Pods` e vocë quase nunca terá que lidar com o
+mapeamento de portas de contêineres para portas do host. Isso cria um modelo simples, 
+retro-compatível onde os `Pods` podem ser tratados muito mais como VMs ou hosts 
+físicos da perspectiva de alocação de portas, nomes, descobrimento de serviços 
+(_service discovery_), balanceamento de carga, configuração de aplicações e migrações.
+
+O Kubernetes impõe as seguintes requisitos fundamentais para qualquer implementação de 
+rede (exceto qualquer política de segmentação intencional):
+   * pods em um nó podem se comunicar com todos os pods em todos os nós sem usar _NAT_.
+   * agentes em um nó (por exemplo o kubelet ou um serviço local) podem se comunicar com 
+     todos os Pods naquele nó.
+
+Nota: Para as plataformas que suportam `Pods` executando na rede do host (como o Linux):
+
+   * pods alocados na rede do host de um nó podem se comunicar com todos os pods
+     em todos os nós sem _NAT_.
+
+Esse modelo não só é menos complexo, mas é principalmente compatível com o 
+desejto do Kubernetes de permitir a portabilidade com baixo esforço de aplicações 
+de VMs para contêineres. Se a sua aplicação executava anteriormente em uma VM, sua VM
+possuía um IP e podia se comunicar com outras VMs no seu projeto. Esse é o mesmo 
+modelo básico.
+
+Os endereços de IP no Kubernetes existem no escopo do `Pod` - contêineres em um `Pod`
+compartilham o mesmo _network namespace_ - incluíndo seu endereço de IP e MAC.
+Isso significa que contêineres que compõem um `Pod` podem se comunicar entre eles 
+através do endereço `localhost` e respectivas portas. Isso também significa que
+contêineres em um mesmo `Pod` devem coordenar a alocação e uso de portas, o que não 
+difere do modelo de processos rodando dentro de uma mesma VM. Isso é chamado de 
+modelo "IP-por-pod".
+
+Como isso é implementado é um detalhe do agente de execução de contêiner em uso.
+
+É possível solicitar uma porta no `Nó` que será encaminhada para seu `Pod` (chamado 
+de _portas do host_), mas isso é uma operação muito específica. Como esse encaminhamento
+é implementado é um detalhe do agente de execução do contêiner. O `Pod` mesmo 
+desconhece a existëncia ou não de portas do host.
+
+## Como implementar o modelo de conectividade do Kubernetes
+
+Existe um número de formas de implementar esse modelo de conectividade. Esse 
+documento não é um estudo exaustivo desses vários métodos, mas pode servir como 
+uma introdução de várias tecnologisa e serve como um ponto de início.
+
+A conectividade no Kubernetes é fornecida através de plugins de 
+{{< glossary_tooltip text="CNIs" term_id="cni" >}}
+
+As seguintes opções estão organizadas alfabeticamente e não implicam preferência por
+qualquer solução.
+
+{{% thirdparty-content %}}
```

Comment:
It'd be good to localize this shortcode too, perhaps in a separate PR?

Reply:
@sftim followup on this. The third party message already exists (you added it!: https://github.com/kubernetes/website/commit/a7734a60a22d77678d880ba25481746588255acd)

But I guess this is that hugo/docsy bug about localization, right?

Reply:
And BTW again, I ran the website locally and it works fine, I guess something entered in master that solved the problem, or maybe my hugo version solve this problem...who knows :D